### ISSUE TITLE
Simple view mode

### ISSUE THREAD

**Issue by WildRikku** [WildRikku/OctoPrint-SpoolManager#1]
I was using FilamentManager before and was pretty overwhelmed by the spool editing dialogue. The UI does not help but the sheer amount of features is also a bit too much for me, even though I see the use for it. I am thinking of a simple mode that targets users who mainly want to track their filament usage.

Simple mode could be switched with a button on the bottom. Default view could be set in settings.

Simple mode will hide:

- flow rate compensation
- all temperature settings
- QR Code
- database id
- serial number
- all dates
- purchased from
- related footnotes

Basically eliminating the upper right and the lower left section, bringing it closer to the easy to use but now abandoned FilamentManager plugin without losing functionality.

In addition, once a spool was started (used > 0), simple mode will hide all weight fields except remaining.

If a new spool is created in simple mode from a template, a warning will appear when the template has fields filled in that are not visible in simple mode but are selected to be copied in the settings.
